build.gradle

```diff
@@ -918,6 +918,7 @@ project(':tools') {
     compile project(':log4j-appender')
     compile libs.argparse4j
     compile libs.jacksonDatabind
+    compile libs.jacksonDatatype
```

Comment:
We should probably add this to other projects that depend on `Jackson` so that we get reasonable behaviour for Java 8 types.